tests/integration.rs

```diff
@@ -223,7 +223,13 @@ impl Drop for ProtocolTester {
     fn drop(&mut self) {
         self.tcp_device.process.kill().unwrap();
         self.unix_device.process.kill().unwrap();
-        fs::remove_file("test_chain_id_priv_validator_state.json").unwrap();
+
+        match fs::remove_file("test_chain_id_priv_validator_state.json") {
+            Err(ref e) if e.kind() != io::ErrorKind::NotFound => {
```

Comment:
Is it expected this might not exist? I encountered this error even after hopping back to master

Reply:
Okay I think yeah you might need to handle the error.